src/test/java/juangonzalesapi/endpoints/Endpoint.java

```diff
@@ -0,0 +1,44 @@
+package juangonzalesapi.endpoints;
+
+import io.restassured.RestAssured;
```

Comment:
This class should be moved to src/main becuase it is the logic of the request api